Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:16.04
+MAINTAINER Stéphane Brunner <stephane.brunner@camptocamp.com>
+
+RUN \
+  apt-get update && \
+  apt-get install --assume-yes --no-install-recommends wget ca-certificates python python-mapnik mapnik-utils gdal-bin libpq5 libgeos-c1v5 fonts-dejavu node-carto osm2pgsql curl unzip && \
+  cd /tmp && \
+  wget https://bootstrap.pypa.io/get-pip.py && \
+  python get-pip.py && \
+  mkdir /fonts && \
+  mkdir /project && \
+  apt-get remove --assume-yes --purge wget ca-certificates && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /root/.cache/*
+
+COPY setup.py /src/
+COPY requirements.txt /src/
+COPY README.rst /src/
+COPY CHANGES.rst /src/
+COPY tilecloud_chain /src/tilecloud_chain
+RUN \
+  apt-get update && \
+  apt-get install --assume-yes --no-install-recommends gcc python-dev libpq-dev libgeos-dev libmapnik-dev && \
+  cd /src && \
+  pip install -r requirements.txt && \
+  pip install . && \
+  apt-get remove --assume-yes --purge gcc python-dev libpq-dev libgeos-dev libmapnik-dev && \
```

Comment:
Are your python packages still working after that?
Apt will remove `libpq5` automatically, I think.